internal/moreos/testdata/fake_func-e.go

```diff
@@ -39,7 +39,7 @@ func main() {
 
 	// Like envoy.Run.
 	waitCtx, waitCancel := context.WithCancel(context.Background())
-	sigCtx, _ := signal.NotifyContext(waitCtx, syscall.SIGINT, syscall.SIGTERM)
+	sigCtx, stop := signal.NotifyContext(waitCtx, os.Interrupt, syscall.SIGTERM)
 	defer waitCancel()
```

Comment:
nit I think we should call `defer waitCancel()` _before_ signal.NotifyContext :D

Reply:
Thanks!